src/modules/main/Main.tsx

```diff
@@ -21,11 +21,15 @@ const TemplateDocumentation = withLazy(() => import('src/modules/template-docume
 const TemplateManagement = withLazy(() => import('src/modules/template-management'));
 
 const Main = ({ match }: RouteChildrenProps) => {
+
+  const [isSidebarOpen, setIsSidebarOpen] = useState(false);
+  const changeSidebarState = () => setIsSidebarOpen(!isSidebarOpen);
+
   return (
-    <div className={csx.main}>
+    <div className={csx.main} style={{gridTemplateColumns: isSidebarOpen ? "150px 1fr" : "50px 1fr"}}>
```

Comment:
Remove this style change - sidebar menu should be fixed / static based on current state.

Reply:
Sidebar is rendered by overlaying